### PR TITLE
fix(button): added inline variation to plain button

### PR DESCRIPTION
**Pull request by mcoker**
fixes https://github.com/patternfly/patternfly-next/issues/2719

### CODE REVIEW

File: src/patternfly/components/Button/examples/Button.md [hunk 1]
```diff
@@ -50,6 +50,10 @@ import './Button.css'
 {{#> button button--modifier="pf-m-inline pf-m-link"}}
   Inline link
 {{/button}}
+
+{{#> button button--modifier="pf-m-inline pf-m-plain"}}
+  Inline link
```
Comment (mattnolting) [inline src/patternfly/components/Button/examples/Button.md]: Should this be an icon?

Comment (mcoker) [reply src/patternfly/components/Button/examples/Button.md]: Yes! Updated that and a couple of other things.

**Comment by christiemolloy** [comment]
Is there no concern about target size for a button now? If not, it could be helpful to update some of the demos where the original button spacing wouldn't push the icon too far left or too far right and not align it with other text and elements.

**Comment by mcoker** [comment]
@christiemolloy we still care about button size, and this would just be for edge cases - but re-reading the original issue, I originally thought this was a custom layout, but it looks like they're running into a bug I filed in react yesterday where plain button actions in the data list aren't wrapped in an element we use to offset the position of the button so it plays nicely with the row height - https://github.com/patternfly/patternfly-react/issues/3860

I could go either way merging this. On the one hand, I can see it being useful in edge cases, though it isn't accessible from the standpoint of the target size and that, unlike the `pf-c-button.pf-m-link.pf-m-inline` variation, there is no additional treatment to indicate the inline variation is clickable (ie, blue color, underline/color change on hover). WDYT?

**Comment by mcoker** [comment]
Closing since the issue in #2719 is a bug with the data list component.

**Pull request closed without merging**